app/views/application/_menu.html.erb

```diff
@@ -15,23 +15,11 @@
         <li class="divider"></li>
         <li class="active"><%= link_to "Settings", user_path(current_user.id) %></li>
       <% else %>
-        <li>
-          <div class="row collapse">
-            <%= form_for :session, url: {action: "create", controller: "sessions"} do |f| %>
-            <div class="large-4 small-8 columns">
-              <%= f.text_field :email, placeholder: "email" %>
-            </div>
-            <div class="large-4 small-8 columns">
-              <%= f.password_field :password, placeholder: "Password" %>
-            </div>
-            <div class="large-3 small-3 columns">
-              <%= f.submit "Log In" %>
-            </div>
-            <% end %>
-          </div>
-        </li>
 
-        <li class="active"><%= link_to "Sign Up", new_user_path %></li>
+        <li class="active"><a href="#" data-reveal-id="login-modal">Log In</a></li>
```

Comment:
Love how this looks and works really like how clean it makes the navbar look. 
Still wondering if there is any other way to deal with those partials? It makes sense how it is working but as we are really using the new method I wonder if there is a more concise way to do it.